eth/ethconfig/config.go

```diff
@@ -167,6 +167,8 @@ type Config struct {
 
 	OverrideOptimismInterop *uint64 `toml:",omitempty"`
 
+	EnableL2Blob bool
```

Comment:
I would add a few comments of like "Enable accepting EIP-4844 BLOBs on L2"

Reply:
Done.